e2e/cypress.json

```diff
@@ -31,6 +33,7 @@
         "resetBeforeTest": false,
         "runLDAPSync": true,
         "secondServerURL": "http://localhost/s/p",
+        "serverEdition": "Team",
```

Comment:
This is not new and added here to be explicit on default. Options care found at https://github.com/mattermost/mattermost-webapp/blob/master/e2e/cypress/utils/constants.js#L11-L13